tcms/testplans/static/testplans/js/get.js

```diff
@@ -127,9 +127,10 @@ function attachEvents(testCases, testPlanId, permissions) {
             }
             const testCaseId = getCaseIdFromEvent(ev);
 
-            // default_tester can be update only if we pass an user id
             jsonRPC('TestCase.update', [testCaseId, {'default_tester': email_or_username}], function(res) {
-                // todo update tc row data
+                const testCaseRow = $(ev.target).closest('.js-testcase-row');
+                testCaseRow.fadeOut(500).fadeIn(500);
```

Comment:
animation looks good. Can you add a top-level constant for the duration and use it everywhere so we can change that easily if we decide to